src/core/Form/FilterInput/FilterInput.tsx

```diff
@@ -15,6 +15,7 @@ import { LabelText, LabelMode } from '../LabelText/LabelText';
 import { HintText } from '../HintText/HintText';
 import { StatusText } from '../StatusText/StatusText';
 import { baseStyles } from './FilterInput.baseStyles';
+import { ReactNode } from 'hoist-non-react-statics/node_modules/@types/react';
```

Comment:
Is this one of VSCode's "smart" auto-imports or on purpose?

Reply:
😬  Yes it is, wonder why it didn't brake the build. Will fix!

Reply:
Fixed.